specification/src/main/asciidoc/platform/TransactionManagement.adoc

```diff
@@ -240,19 +240,19 @@ access a database, invoke an enterprise bean that accesses the same
 database as part of the same transaction, and, finally, commit the
 transaction.
 
-A Java EE product must provide a transaction
+A Jakarta EE product must provide a transaction
 manager that is capable of coordinating two-phase commit operations
 across multiple XA-capable JDBC databases. If a JDBC driver supports the
 Java Transaction API’s XA interfaces (in the _javax.transaction.xa_
-package), then the Java EE product must be capable of using the XA
+package), then the Jakarta EE product must be capable of using the XA
 interfaces provided by the JDBC driver to accomplish two-phase commit
-operations. The Java EE product may discover the XA capabilities of JDBC
+operations. The Jakarta EE product may discover the XA capabilities of JDBC
 drivers through product-specific means, although normally such JDBC
 drivers would be delivered as resource adapters using the Connector API.
 
 === [[a520]]Transactional JMS Support
 
-A Java EE product must support a JMS provider
+A Jakarta EE product must support a JMS provider
 as a transactional resource manager. The platform must enable
 transactional JMS access from servlets, JSP pages, and enterprise beans.
```

Comment:
Change JSP to "server pages" (generalize the use)

Reply:
Resolved